### PR TITLE
Added alternate instructions on how to sign up to our mailing lists

### PR DESCRIPTION
**Pull request by bmispelon**
This should make it possible for people who can't access
groups.google.com (blocked in China) to sign up anyway.

### CODE REVIEW

**Comment by bmispelon** [comment]
Looks like this:
django_community_django_-_2015-12-04_16 03 44

File: djangoproject/templates/includes/mailing_lists.html [hunk 1]
```diff
@@ -9,6 +9,13 @@
         <span class="visuallyhidden">Subscribe</span>
       </button>
     </form>
+    <p class="meta">
+        If for some reason you cannot access
+        <a href="https://groups.google.com">groups.google.com</a>, you can subscribe
+        directly by sending an email to
```
Comment (timgraham) [inline djangoproject/templates/includes/mailing_lists.html]: Looks good. I'd omit "directly". By the way, some information is duplicated at https://docs.djangoproject.com/en/dev/internals/mailing-lists/

**Comment by bmispelon** [comment]
I added a paragraph linking to the docs for more mailing lists:
django_community_django_-_2015-12-04_16 40 26

File: djangoproject/templates/includes/mailing_lists.html [hunk 2]
```diff
@@ -9,6 +9,13 @@
         <span class="visuallyhidden">Subscribe</span>
       </button>
     </form>
+    <p class="meta">
+      If for some reason you cannot access
```
Comment (timgraham) [inline djangoproject/templates/includes/mailing_lists.html]: Didn't realize this displays on the homepage. Maybe "You can also subscribe by..." would be enough?
"and following..."

File: djangoproject/templates/includes/mailing_lists.html [hunk 3]
```diff
@@ -20,5 +27,19 @@
...
+      <a href="https://groups.google.com">groups.google.com</a>, you can subscribe
+      by sending an email to
+      <a href="mailto:django-developers+subscribe@googlegroups.com">django-developers+subscribe@googlegroups.com</a>
+      and follow the instructions that will be emailed to you.
+    </p>
   </div>
 </div>
+<div class="layout-2col"><!-- separate div so I can reuse the margin-top -->
```
Comment (timgraham) [inline djangoproject/templates/includes/mailing_lists.html]: <p> has a top-margin -- it looks fine without the div to me?

**Pull request merged**